src/main/scala/io/kaizensolutions/virgil/PullMode.scala

```diff
@@ -0,0 +1,8 @@
+package io.kaizensolutions.virgil
+
+sealed trait PullMode
+object PullMode {
```

Comment:
Maybe possible to hide this since you have `take` on CQL.

Reply:
Hidden